components/avatar/Avatar.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class Avatar extends PureComponent {
+  static propTypes = {
+    borderColor: PropTypes.oneOf([ 'neutral', 'mint', 'aqua', 'violet', 'teal', 'gold', 'ruby', 'tealdark' ]),
+    className: PropTypes.string,
+    counter: PropTypes.element,
+    image: PropTypes.string,
+    size: PropTypes.oneOf([ 'tiny', 'small', 'medium' ]),
+  };
+
+  static defaultProps = {
+    borderColor: 'white',
+    size: 'medium',
+  };
+
+  render () {
+    const {
+      borderColor,
+      className,
+      counter,
+      image,
+      size,
+      ...others
+    } = this.props;
+
+    const avatarClasses = cx(
+      theme.avatar,
+      theme[size],
+      className,
+    );
+
+    const imageClasses = cx(
+      theme.image,
+      theme[`border-${borderColor}`],
+    );
+
+    return (
+      <div className={avatarClasses} {...others} data-teamleader-ui="avatar">
+        <div className={imageClasses}>
+          <img src={image} />
```

Comment:
It would be nice to have an `alt` tag here, the name of the avatar or something like that.